Tests/SentryTests/Integrations/OutOfMemory/SentryOutOfMemoryIntegrationTests.swift

```diff
@@ -23,7 +62,35 @@ class SentryOutOfMemoryIntegrationTests: XCTestCase {
         XCTAssertEqual(path, ProcessInfo.processInfo.environment["XCTestConfigurationFilePath"])
     }
     
+#if os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)
+    func testANRDetected_UpdatesAppStateToTrue() {
```

Comment:
Nice tests 😁, but I think I added them on a previous PR.

Reply:
Now we double check it 😁

After merging it I will remove any duplication.